src/components/About/Blog/BlogLoader.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ContentLoader from 'react-content-loader';
+import styled from 'styled-components';
+
+const Center = styled.div`
+  display: block;
+  position: relative;
+
+  * {
+    position: relative;
+    left: 15%;
+    margin-top: 30px;
+  }
+`;
+
+const SingleLoader = () => (
```

Comment:
Rename to `BlogCardLoader` or `BlogCardSkeleton`

Reply:
Done Sir!